src/ARM.cpp

```diff
@@ -104,7 +106,7 @@ const u32 ARM::ConditionTable[16] =
     0x0000  // NE
 };
 
-ARM::ARM(u32 num, ARMJIT::ARMJIT& jit, Melon::GPU& gpu) :
+ARM::ARM(u32 num, ARMJIT& jit, melonDS::GPU& gpu) :
```

Comment:
why do you prefix the GPU?

Reply:
Because having a member with the same name as a type causes C++ to choke, and naming the type as `class GPU` looks weird.